src/EditorFeatures/Test2/FindReferences/FindReferencesTests.NamedTypeSymbols.vb

```diff
@@ -2156,5 +2156,35 @@ public class D { }
 
             Test(input)
         End Sub
+
+        <Fact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        <WorkItem(1174256)>
+        Public Sub TestFarWithInternalsVisibleToNull()
+            Dim input =
+<Workspace>
+    <Project Language="C#" AssemblyName="ClassLibrary1" CommonReferences="true">
+        <Document><![CDATA[
+        [assembly: System.Runtime.CompilerServices.InternalsVisibleTo(null)]
+        internal class A
+        {
+        }]]>
+        </Document>
+    </Project>
+
+    <Project Language="C#" AssemblyName="ClassLibrary2" CommonReferences="true">
+        <ProjectReference>ClassLibrary1</ProjectReference>
+        <Document><![CDATA[
+        public class B
+        {
+            [$$A] _a;
```

Comment:
Possibly. I tried a few combinations. It doesn't actually matter if the 2nd project compiles, just that it tries to reference something unseen that will exercise the DependentProjectsFinder code (I verified by putting a BP on the new null check). You might have to help me write a suitable test.